### ISSUE TITLE
Strikethrough text and accessibility

### ISSUE THREAD

**Issue by ssilvio10** [wet-boew/wet-boew-styleguide#419]
Hello, the guidance on strikethrough, doesn't indicate that invisible text should be used.  Not all screen readers are able read the strikethrough element, so the recommendation would be to include invisible text to ensure all users get the same information.  Refer to:  https://www.webaxe.org/strikethrough-html-accessibility/.  I am hoping this guidance, could be added to the Web Experience Toolkit, compliance points for text editing effects.
